lib/entice/logic/maps/map.ex

```diff
@@ -52,7 +52,8 @@ defmodule Entice.Logic.Map do
 
 
   defp content(mod) do
-    name = mod |> Module.split |> List.last |> to_string
+    [map_atom] = elem(mod,2)
```

Comment:
We get the unquoted mapname as far as I understand which looks something like [something, something, [:Lobby]].
So we get element at index 2, [:Lobby] and extract the atom from the list.